### PR TITLE
Make it a monorepo

### PR DESCRIPTION
**Pull request by c0rydoras**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# CHANGELOG
+
+This has been discontinued as it is redundand. The new Changelog starting with v4 can be viewed [here](https://github.com/adfinis/timed/releases/)
```
Comment (hairmare) [inline CHANGELOG.md]:
```suggestion
This has been discontinued as it is redundant. The new Changelog starting with v4 can be viewed [here](https://github.com/adfinis/timed/releases/)
```

File: LICENSE [hunk 2]
```diff
@@ -629,8 +629,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    timed-backend.src
-    Copyright (C) 2016  ad-sy
+    timed
+    Copyright (C) 2024 Adfinis AG
```
Comment (hairmare) [inline LICENSE]: feel free to replace this file with an unaltered copy from upstream

as the para above this states:

> attach the following notices to the program

so like in a README or similar, not by changing anything in LICENSE itself

File: .github/dependabot.yml [hunk 3]
```diff
@@ -26,11 +26,22 @@ updates:
           - "pytest-*"
 
   - package-ecosystem: docker
-    directory: "/"
+    directory: "/backend"
```
Comment (hairmare) [inline .github/dependabot.yml:29]: the frontend probably also needs docker

**Review by hairmare — COMMENTED** [review]
looks promising

merging the helm chart down the line will enable a whole new class of e2e testing

File: LICENSE (hunk 2 above)
Comment (c0rydoras) [reply LICENSE]: döner

File: .github/dependabot.yml (hunk 3 above)
Comment (c0rydoras) [reply .github/dependabot.yml:29]: done

**Comment by c0rydoras** [comment]
this is outdated, please review #14

**Pull request closed without merging**